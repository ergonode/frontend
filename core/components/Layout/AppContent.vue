/*
 * Copyright Â© Bold Brand Commerce Sp. z o.o. All rights reserved.
 * See LICENSE for license details.
 */
<template>
    <main class="app-content">
        <slot />
    </main>
</template>

<script>
export default {
    name: 'AppContent',
};
</script>

<style lang="scss" scoped>
    .app-content {
        display: flex;
        flex: 1;
        overflow: hidden;
        margin-top: 48px;
    }
</style>
