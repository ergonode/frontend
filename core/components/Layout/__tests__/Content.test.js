/*
 * Copyright Â© Bold Brand Commerce Sp. z o.o. All rights reserved.
 * See LICENSE for license details.
 */
import { shallowMount } from '@vue/test-utils';
import AppContent from '~/core/components/Layout/AppContent.vue';


describe('Layout/AppContent', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallowMount(AppContent);
    });

    it('Component is named well', () => {
        expect(typeof AppContent.name).toBe('string');
        expect(AppContent.name).toEqual('AppContent');
        expect(wrapper).toMatchSnapshot();
    });
});
