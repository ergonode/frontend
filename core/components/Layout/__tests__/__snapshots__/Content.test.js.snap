// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout/AppContent Component is named well 1`] = `<main class="app-content"></main>`;
